projects/plugins/jetpack/functions.opengraph.php

```diff
@@ -593,7 +597,20 @@ function jetpack_add_fediverse_creator_open_graph_tag( $tags ) {
 
 		// services can have multiple connections. Store them all in our array.
 		foreach ( $connections as $connection ) {
-			$connection_id   = $publicize->get_connection_id( $connection );
+			$connection_id = $publicize->get_connection_id( $connection );
```

Comment:
Is this solving the problem? I think we want to get away from using the connections data at all as the is the thing that can trigger the API request from the `get_services` call above. Instead can we check the service in the `_publicize_shares` meta?

Reply:
I thought the idea here is to use the meta so we only add the tags to posts where we actually shared to, not everywhere if the user has a Mastodon connection. 🤔 

> Instead can we check the service in the _publicize_shares meta

We cannot fully move away from it unless we make some other changes on the backend, because we only store the `display_name` there, not the handle

Reply:
This way we decrease the number of API calls, as we return early if there was no shares made to a post.

Reply:
Mmm, I might be confusing the issue then, but the suggestion was to prevent getting the active connections here:

https://github.com/Automattic/jetpack/blob/69b12fd1f2d4c91ca7555de3403f5080e984a592/projects/plugins/jetpack/functions.opengraph.php#L589

Which I think we're still doing? We also currently always output this whether the Mastodon connection has been shared to or not, so returning early is a change in behaviour. I'm not sure if that's a problem or not, but I think we would want these tags if someone else shares a link to the post on Mastodon.

Reply:
> the suggestion was to prevent getting the active connections here:

Yeah, I think this is a call we should avoid doing if possible. If we can adapt what we store in post meta to make that possible, that seems like a good tradeoff to me.